k-distribution/src/main/scripts/lib/k.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+execute () {
+  (
+  if $verbose; then
+    set -x
+  fi
+  "$@"
+  )
+}
+
+output () {
+  target="$1"
+  shift
+  if [ "$target" = "/dev/stdout" ]; then
+    "$@"
+  elif [ "$target" = "/dev/stderr" ]; then
+    "$@" 1>&2
+  else
+    "$@" > "$target"
+  fi
+}
+
+catNewline () {
+  cat "$@"
+  echo
+}
```

Comment:
Please add some comments to describe why this file exists and what it contains.
Basically what you just wrote in the PR description, but add it here so it's easily accessible in the future.

Reply:
Well it's also worth it to mention at the top of the file that this file is for utilities used across all our scripts, and so any common bash functionalities should be factored out to here.

Reply:
Maybe rename this file to `utils` or `lib`?